### ISSUE TITLE
PC cannot find any services started from Android

### ISSUE THREAD

**Issue by evbarnett** [jmdns/jmdns#169]
I want a PC and Android, each running JmDNS, to find each other

Currently,

Android can find PC
Android can find Android
PC can find PC
BUT, PC cannot find Android

Any ideas? I've tried everything and I cannot get the PC to find it.

**Comment by nikialeksey** [comment]
Hey!

Have you solved it?

**Comment by evbarnett** [comment]
Yes I had solved it, but I can't quite remember what I did since it's been a while. I *think* the solution was I had to open the port manually on the PC side

**Comment by nikialeksey** [comment]
Thank you very much!

Did not think you will answer, the issue is too old... Yes, I'm also trying to play with firewall, will try more

**Comment by evbarnett** [comment]
Later this week I can check through some old code to see if I can find the solution I reached. I'll let you know if I find anything
